modules/user_accounts/php/edit_user.class.inc

```diff
@@ -441,7 +440,7 @@ class Edit_User extends \NDB_Form
                             WHERE userID=:uid",
                     ['uid' => $uid]
                 );
-            } elseif (!empty($examinerID)
+            } elseif (!count($examinerID) !== 0
```

Comment:
This logic is confusing. Did you mean `count(examinerID) > 0`?

Reply:
Sorry yeah, missed removal of the ! at the start. I'll change to > 0 though, I like that better